.changeset/nasty-jars-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/s3reader-adapter': major
```

Comment:
Could the name of the EA have csv in it somewhere, if it is purely about csv parsing from s3?